src/js/components/Anchor/__tests__/__snapshots__/Anchor-test.tsx.snap

```diff
@@ -464,8 +464,8 @@ exports[`Anchor matches icon size to size prop when theme.icon.matchSize is true
   -webkit-flex: 0 0 auto;
   -ms-flex: 0 0 auto;
   flex: 0 0 auto;
-  width: NaNpx;
-  height: NaNpx;
+  width: NaNxsmall;
```

Comment:
do we want these? I remember they are from `grommet-icons` wasn't sure if we looked into why they changed

Reply:
I'm guessing it is coming from this PR, but not positive: https://github.com/grommet/grommet-icons/pull/325/files
Either way I think visually this test wouldn't change. The width is just being replaced with another invalid option.
At some point we should look at the `useSizedIcon` function in Grommet, I think this is where the issue may be coming from. I think the icons are supposed to scale to the size of the Anchor. This isn't working currently and this PR doesn't change that it just changes the incorrect value to another incorrect value.
<img width="195" alt="Screen Shot 2024-05-14 at 9 00 58 AM" src="https://github.com/grommet/grommet/assets/54560994/de13cb27-410c-4789-9a59-f413154fc478">

Reply:
Do we have a ticket somewhere to capture?

Reply:
Issue created: https://github.com/grommet/grommet/issues/7231